README.md

```diff
@@ -15,8 +15,8 @@ Starter kits should include a set of frameworks, libraries, and essential toolin
 
 Starter kits should include some core libraries and technologies that a new project will need. These pieces should come configured according to best practices and be ready to use togther as a unit.
 
-- Framework?: (Next, Nuxt, SvelteKit, Remix, etc)
-- UI: (react, vue, angular, etc)
```

Comment:
You don't need to change this but I am adding my protest via comment. Maybe we need different terminology besides Framework and UI.  Maybe Meta Framework and Framework? I think vue, angular, and remix should be grouped together it's super weird that React is by itself here. cc @dustinsgoodman

Reply:
Agree

Reply:
@danecando Yeah, I was having a lot of issues here. I think eventually we'll include Qwik, Svelte and Solid which fall in the UI Library section as they aren't fully featured frameworks. But Angular and Vue are as fully featured as Remix out of the box making this a hard debate. It's definitely a good discussion point.

Reply:
I know that there is a difference using React as the example where it was always marketed as the V in MVC and just handled the UI concerns but I still think it would be easier from a user perspective to categorize the core frameworks together or even double categorize some of them. IE Angular could be part of Framework and UI Frameworks/Libraries. Or we can call them something like Meta Frameworks / Web Frameworks. Not sure what is the best solution but I think there's something that needs to be done with this

Reply:
So whats the conclusion, should i use replace Framework with Meta Framework and UI -> Framework ? @dustinsgoodman  @danecando

Reply:
Let's go "Core UI" and "Frameworks" for now. Still debating but I think it's beyond the scope of this change to finish this debate.

Reply:
cc @kodejuice